docs/admin/configure.md

```diff
@@ -149,6 +150,9 @@ CODER_PG_CONNECTION_URL=
 # Boolean. Specifies if TLS will be enabled.
 CODER_TLS_ENABLE=
 
+# If CODER_TLS_ENABLE=true && Opening access to outside or want to change TLS port:
```

Comment:
We should re-word this to be:

> # If CODER_TLS_ENABLE=true and you want to change the TLS port:

Reply:
Yes, the main reason I added this was that I had enabled TLS but was still unable to access :3443 using the IP or URL from the outside. After an hour of digging through documentation and the CLI --help prompts, I found a flag for `CODER_TLS_ADDRESS`. I tried adding this to the .env file with 0.0.0.0 and suddenly everything was working. I think it's really important that configuration settings that one would most likely be using be mentioned in the documentation instead of being in a dense CLI prompt.